src/lua/log.lua

```diff
@@ -650,7 +654,10 @@ local log = {
         cfg_set_log_format = box_api_set_log_format,
     },
     internal = {
-        ratelimit = Ratelimit
+        ratelimit = {
+            Ratelimit = Ratelimit,
+            cfg = ratelimit_cfg,
+        },
```

Comment:
I don't like the internal API:
1. `box.internal.ratelimit.Ratelimit`: the 'ratelimit' word is repeated. Also, we don't typically Capitalize class/module names in Lua.
2. `box.internal.ratelimit.cfg.enable = true` sounds like it should enable rate limiting, but in fact it disables it.
3. `Ratelimit:new()` should be a function of the ratelimit module, not a method of the ratelimit class.

I suggest to rework it as follows:
1. To create a rate limiter, use `box.internal.ratelimit.new()` (note that a dot (.) is used, not a colon (:))
2. To disable/enable rate limiting globally, use `box.internal.ratelimit.disable()` and `box.internal.ratelimit.enable()`.

Reply:
Fixed: changed the API accordingly.